packages/terra-status-view/src/StatusView.jsx

```diff
@@ -0,0 +1,210 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import Divider from 'terra-divider';
+import styles from './StatusView.scss';
+
+const cx = classNames.bind(styles);
+
+const variants = {
+  NODATA: 'no-data',
```

Comment:
We should seperatet these with hyphens to be readable.

Reply:
I prefer leaving the hypens out for when variables are in all caps

Reply:
Whatever we do we do for other constants we attach to `Opts`, we should be consistent what that.